deployment/gem.rb

```diff
@@ -235,7 +235,7 @@ def ask(question, default)
   task :migrate do
     status = capture(%{
       cd #{latest_release} &&
-      php app/console --env=prod doctrine:migrations:migrate --dry-run --no-interaction
+      #{php_bin} app/console --env=prod doctrine:migrations:migrate --dry-run --no-interaction
```

Comment:
it shouldn't since it is always by default set to php
https://github.com/sumocoders/forkcms_deploy/blob/0040d78c13bf312fbb0d2a87f3f55949d7683620/lib/forkcms_deploy/defaults.rb#L19